### PR TITLE
Weather APP - JS branch Feature v1

### PR DESCRIPTION
**Pull request by brugobi**
## Weather app

Here is a solo weather app project building with JS and API

### Features

- [X] Get weather information by getting the user location if the user accepts it;

- [X] Get weather information by typing the city's name;

-  [X] Get the image as background by city's name;

- [X] Autocomplete functionality for cities names;

- [X] Temperature available in Celsius and Fahrenheit by clicking on the temperature number;

### Thanks for reviewing my code

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              <div>
+                <p id="btnTemp" class="temperature temp-celsius has-text-warning is-size-1">?°C</p>
+              </div>
+              <div>
+                <p class="location has-text-light is-size-5">location, country</p>
+              </div>
+            </div>
+            <div class="column">
+              <div class="icon-div">
+                <div class="weather-icon"><img src="icons/unknown.png" alt=""></div>
```
Comment (mosaaleb) [inline src/index.html]: - [x] There is an console error coming from the image source. Let's fix the used image path
Screenshot from 2020-12-15 00-20-03

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    weather.temp_min = Math.floor(data.main.temp_min - KELVIN);
+    weather.temp_max = Math.floor(data.main.temp_max - KELVIN);
+    weather.description = data.weather[0].description;
+    weather.iconId = data.weather[0].icon;
+    weather.city = data.name;
+    weather.country = data.sys.country;
+    displayWeather();
+  } catch (err) {
+    // eslint-disable-next-line no-console
+    console.log(err);
```
Comment (mosaaleb) [inline src/index.js]: - [x] No `console.log` commited to repo. It kills performance.



We can replace the error handling by displaying the information in a notification box.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+// app data
+const weather = {};
+weather.temperature = {
+  unit: false,
+};
+
+const KELVIN = 273;
+const KEY = '15b72f8181c849f71bb8b90b88730574';
+
+const displayWeather = () => {
```
Comment (mosaaleb) [inline src/index.js]: - [x] Since this function is a DOM-related, maybe we should move it to `dom.js`?

**Review by mosaaleb — CHANGES_REQUESTED** [review]

**Comment by brugobi** [comment]
- Updated the code adding all the recommendations requested;

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval. 
**Nice UI concept also. 👌**

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**